### ISSUE TITLE
ReflectionTypeLoadException error occur if moving all the exts to "Extensions" folder

### ISSUE THREAD

**Issue by Smilefounder** [ExtCore/ExtCore-Sample-Modular-Ui-Adv#1]
System.Reflection.ReflectionTypeLoadException: 'Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.'

![image](https://cloud.githubusercontent.com/assets/3785721/26272570/d3927522-3d45-11e7-8de0-deaa18bab274.png)

**Comment by DmitrySikorsky** [comment]
Hi! I'm almost sure you have this problem because some of your extension dlls has reference to the assembly that the main web application doesn't have reference to AND the extensions folder doesn't contain.

**Comment by Smilefounder** [comment]
@DmitrySikorsky : thanks for your answer. Let me try again 🥇

**Comment by DmitrySikorsky** [comment]
I'm closing it for now, feel free to re-open it any time if have any additional questions.
